src/cts/src/HTreeBuilder.h

```diff
@@ -158,7 +161,7 @@ class HTreeBuilder : public TreeBuilder
     unsigned getRemainingLength() const { return remainingLength_; }
 
    private:
-    const double length_;
+    double length_;  // xxx remove const
```

Comment:
Why the comment - is there some uncertainty here?  No xxx needed

Reply:
Fixed